core/domain/services/admin/privacy/forms/PrivacySettingsFormHandler.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\privacy\forms;
+
+use EE_Config;
+use EE_Form_Section_HTML;
+use EE_Form_Section_Proper;
+use EE_Full_HTML_Validation_Strategy;
+use EE_Registration_Config;
+use EE_Registry;
+use EE_Select_Reveal_Input;
+use EE_Text_Area_Input;
+use EEH_HTML;
+use EventEspresso\core\exceptions\InvalidFormSubmissionException;
+use EventEspresso\core\libraries\form_sections\form_handlers\FormHandler;
+use EventEspresso\core\services\loaders\LoaderFactory;
+
+/**
+ * Class PrivacySettingsFormHandler
+ *
+ * Class for displaying and processing the privacy settings form
+ *
+ * @package        Event Espresso
+ * @author         Mike Nelson
+ * @since          $VID:$
+ *
+ */
+class PrivacySettingsFormHandler extends FormHandler
+{
+
+    /**
+     * @var EE_Config
+     */
+    protected $config;
+
+
+    /**
+     * PrivacySettingsFormHandler constructor.
+     *
+     * @param EE_Registry $registry
+     * @param EE_Config   $config
+     */
+    public function __construct(EE_Registry $registry, EE_Config $config)
+    {
+        $this->config = $config;
+        parent::__construct(
+            esc_html__('Privacy Settings', 'event_espresso'),
+            esc_html__('Privacy Settings', 'event_espresso'),
+            'privacy-settings',
+            '',
+            FormHandler::DO_NOT_SETUP_FORM,
+            $registry
+        );
+    }
+
+
+    /**
+     * creates and returns the actual form
+     *
+     * @return EE_Form_Section_Proper
+     */
+    public function generate()
+    {
+        /**
+         * @var $reg_config EE_Registration_Config
+         */
+        $reg_config = $this->config->registration;
+        return new EE_Form_Section_Proper(
+            array(
+                'name'        => 'privacy_consent_settings',
+                'subsections' => array(
+                    'privacy_consent_form_hdr' => new EE_Form_Section_HTML(
+                        EEH_HTML::h2(esc_html__('Privacy Policy Consent Settings', 'event_espresso'))
+                    ),
+                    'enable'                   => new EE_Select_Reveal_Input(
+                        array(
+                            'enable-privacy-consent' => esc_html__('Enabled', 'event_espresso'),
+                            'disable'                => esc_html__('Disabled', 'event_espresso'),
+                        ),
+                        array(
+                            'default'         => $reg_config->isConsentCheckboxEnabled() ? 'enable-privacy-consent'
+                                : 'disable',
```

Comment:
formatting looks funky here.. `? 'enable-privacy-consent'` should be on its own line

Reply:
d'oh! right